tests/ui/panics/runtime-switch.rs

```diff
@@ -18,9 +18,9 @@
 //@ ignore-android FIXME #17520
 //@ ignore-openbsd no support for libbacktrace without filename
 //@ ignore-wasm no backtrace support
-//@ ignore-emscripten no panic or subprocess support
-//@ ignore-sgx no subprocess support
+//@ ignore-emscripten no panic support
```

Comment:
isn't this also "no backtrace support"? It's a panic=abort target anyway, and this is not an unwind test

Reply:
Yeah I wasn't sure about that one, which is why I opened [Cleanup "no panic support" tests #135923](https://github.com/rust-lang/rust/issues/135923). I have no idea what "no panic support" actually means, so I didn't modify *those* ones in this PR.